rholang/src/test/scala/coop/rchain/rholang/interpreter/NormalizeTest.scala

```diff
@@ -543,6 +543,39 @@ class ProcMatcherSpec extends FlatSpec with Matchers {
       ProcNormalizeMatcher.normalizeMatch(pMatch, inputs)
     }
   }
+  "PMatch" should "Handle a match inside a for pattern" in {
+    // for (@(match (x | y) { case 47 => Nil }) <- @Nil) { Nil }
```

Comment:
Can you write the example according to the grammar?
match {x | y} {
  47 => Nil
}

Reply:
Done